vscode-antimony/src/server/virtualEnvWin.bat

```diff
@@ -4,7 +4,7 @@ echo "script runs"
 
 setlocal
 set py=%1
-if "%py%"=="" set py=python
+if "%py%"=="" set py=python3.11
```

Comment:
Dr. Sauro mentioned that the Python command for Windows is just python as opposed to python[version]. Are you sure this will work?